package.json

```diff
@@ -1,138 +1,157 @@
-    "@types/lodash": "^4.14.149",
-    "@types/nanoid": "^2.1.0",
-    "@types/node": "^13.1.6",
-    "@types/react": "^16.9.17",
-    "@types/react-dom": "^16.9.4",
-    "@types/react-redux": "^7.1.5",
-    "@types/react-router-dom": "^5.1.3",
-    "@types/redux-mock-store": "^1.0.1",
-    "@types/yup": "^0.26.27",
-    "@typescript-eslint/eslint-plugin": "^2.15.0",
-    "@typescript-eslint/parser": "^2.15.0",
-    "cross-env": "^6.0.3",
-    "cssnano": "^4.1.7",
-    "dayjs": "^1.8.19",
-    "enzyme": "^3.11.0",
-    "enzyme-adapter-react-16": "^1.15.2",
-    "enzyme-to-json": "^3.4.3",
-    "eslint": "^6",
-    "eslint-config-airbnb": "^18.0.1",
-    "eslint-config-prettier": "^6",
-    "eslint-plugin-import": "^2.19.1",
-    "eslint-plugin-jsx-a11y": "^6.2.1",
-    "eslint-plugin-node": "^11.0.0",
-    "eslint-plugin-prettier": "^3.1.2",
-    "eslint-plugin-promise": "^4.1.1",
-    "eslint-plugin-react": "^7.17.0",
-    "eslint-plugin-react-hooks": "^1.7.0",
-    "eslint-plugin-testcafe": "^0.2.1",
-    "fetch-mock": "^8.3.1",
-    "jest": "^24.9.0",
-    "jest-junit": "^10.0.0",
-    "laravel-mix": "^5.0.1",
-    "nanoid": "^2.1.9",
-    "node-sass": "^4.13.0",
-    "prettier": "^1.19.1",
-    "react-docgen-typescript-loader": "^3.6.0",
-    "react-intl-translations-manager": "^5.0.3",
-    "redux-mock-store": "^1.5.4",
-    "resolve-url-loader": "^3.1.1",
-    "sass-loader": "^7.3.1",
-    "storybook-addon-intl": "^2.4.1",
-    "testcafe": "^1.7.1",
-    "ts-jest": "^24.3.0",
-    "ts-loader": "^6.2.1",
-    "typescript": "^3.7.4",
-    "vue-template-compiler": "^2.6.11",
-    "webpack": "^4.41.5"
-  },
-  "dependencies": {
-    "@formatjs/intl-pluralrules": "^1.5.0",
-    "@formatjs/intl-relativetimeformat": "^4.5.7",
-    "axios": "^0.19.1",
-    "babel-plugin-react-intl": "^5.1.16",
-    "classnames": "^2.2.6",
-    "clone-framework": "^2.4.12",
-    "core-js": "^3",
-    "formik": "^1.5.8",
-    "history": "^4.10.1",
-    "intl": "^1.2.5",
-    "isomorphic-fetch": "^2.2.1",
-    "jquery": "^3.4.0",
-    "lodash": "^4.17.15",
-    "nprogress": "^0.2.0",
-    "re-reselect": "^3.4.0",
-    "react": "^16.12.0",
-    "react-dom": "^16.12.0",
-    "react-intl": "^3.11.0",
-    "react-redux": "^7.1.3",
-    "react-router-dom": "^5.1.2",
-    "react-test-renderer": "^16.12.0",
-    "redux": "^4.0.5",
-    "redux-api-middleware": "^3.1.0",
-    "redux-devtools-extension": "^2.13.8",
-    "redux-logger": "^3.0.6",
-    "redux-thunk": "^2.3.0",
-    "reselect": "^4.0.0",
-    "sweetalert2": "^9.5.4",
-    "universal-router": "^8.3.0",
-    "yup": "^0.28.0"
-  },
-  "jest-junit": {
-    "outputDirectory": "./report/jest",
-    "outputName": "junit.jest.xml"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/GCTC-NTGC/TalentCloud.git"
-  },
-  "bugs": {
-    "url": "https://github.com/GCTC-NTGC/TalentCloud/issues"
-  },
-  "homepage": "https://github.com/GCTC-NTGC/TalentCloud#readme"
+    "private": true,
```

Comment:
In my settings:
```
"[json]": {
        "editor.defaultFormatter": "vscode.json-language-features",
        "editor.formatOnSave": true
    },
```
I usually save configs without formatting though, to avoid this. My bad.